frontend/src/assets/i18n/en.json

```diff
@@ -282,6 +282,17 @@
                 "legend-desc": "The image shows the relative difference in total cumulative impact between the base and what-if scenario."
             }
         },
+        "histogram": {
+            "caption": "Histogram",
+            "interval": "Interval",
+            "zeroes-included": "Full range",
+            "zeroes-excluded": "NB. The histogram excludes zero values",
+            "include-zero": "Include zero",
+            "exclude-zero": "Exclude zero",
+            "cumulative-proportion": "Cumulative proportion",
+            "cumulative-proportion-exclusive": "Cum. proportion w/o zeroes",
+            "median-in-range": "Median in range"
```

Comment:
Directly after merging this PR, I realize that "median" is a misleading term to use (not to mention, plain wrong :confused: ).
I'll take care of it.

Reply:
Should be "middle" or something? I'll try and find something that'll fit.